cmd/cmd.go

```diff
@@ -43,8 +43,10 @@ const (
 	FlagStatusAddr = "status-addr"
 	// FlagSlowLogFile is the name of slow-log-file flag.
 	FlagSlowLogFile = "slow-log-file"
-	// FlagRedactLog is whether to redact sensitive information in log.
-	FlagRedactLog = "redact-info-log"
+	// FlagRedactLog is whether to redact sensitive information in log, already deprecated by FlagRedactInfoLog
+	FlagRedactLog = "redact-log"
```

Comment:
Could you add a test in tests/br_other?

Reply:
addressed in https://github.com/pingcap/br/pull/692/commits/3fd299bf1e3e49da29dad8728b5d782e7672c48b